### PR TITLE
init

### PR DESCRIPTION
**Pull request by tostrowski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
-# stp-blog
+Are you looking for a development process for a small team that is oriented to ship code in a fast and robust way? Check out who we do this in Schibsted Media Platform team!
```
Comment (kubek2k) [inline README.md]: s/who/how/ ?

File: README.md [hunk 2]
```diff
@@ -1 +1,51 @@
...
+
+
+Most of the projects that are about to start need to figure out what technologies to use and what processes to put in place to achieve their goals. In many cases, to gain better control, organizations try to enforce some uniform way of working for all the teams. In such case you may upfront inherit a set of processes that not always fit your well. Same with technologies. Licences for various software that company owns or expensive hardware that was bought may impact the way team works and organize itself. But what if you could start from zero. What if you have a team of let's say 6 with and a goal to ship fast high quality software, and you may choose whatever process and tools you like by yourself. That’s exactly the situation that we were about 3 years ago. In terms of technologies, processes and culture we were building from the ground up having in mind principles like simplicity, transparency and trust. In this article we will focus on the development process and deployment pipeline that we use currently in our daily work. To make it more digestible I’ll break the article into three parts.
```
Comment (mateusz-fiolka) [inline README.md]: > to ship fast high quality software

to ship hiqh quality software fast

Comment (mateusz-fiolka) [inline README.md]: > But what if you could start from zero.

But what if you could start from zero?

Comment (mateusz-fiolka) [inline README.md]: > That’s exactly the situation that we were about 3 years ago

we were in?

**Pull request merged**